src/pages/SupplierInvoicePage.js

```diff
@@ -137,12 +248,20 @@ export class SupplierInvoicePage extends React.Component {
     const { database, transaction } = this.props;
     database.write(() => {
       const transactionItem = createRecord(database, 'TransactionItem', transaction, item);
-      createRecord(
+      const transactionBatch = createRecord(
         database,
         'TransactionBatch',
         transactionItem,
         createRecord(database, 'ItemBatch', item, '')
       );
+
+      if (item.isVaccine) {
+        transactionBatch.location = database
```

Comment:
i think it's safe to store default fridge in this.defaultFridge ?

Reply:
Done - 4b4fdc92eb3230c3b32898fa188526581e579061